spec/stepmod/converters/eqn_spec.rb

```diff
@@ -40,7 +40,7 @@
       XML
     end
     let(:output) do
-      "[[eqnGM1]]\n\n[stem]\n++++\nχ ms = V - E + 2F - L l - 2(S - G s) = 0\n++++\n\n"
+      "[[eqnGM1]]\n\n[stem]\n++++\nχ _{ms} = V - E + 2F - L _{l}  - 2(S - G ^{s})  = 0\n++++\n\n"
```

Comment:
Can we use heredoc/multi-line strings for comparison?

Reply:
Done